File: backend/src/main/java/isep/webapp/terasync/service/FaqService.java
package isep.webapp.terasync.service;

import isep.webapp.terasync.model.Faq;
import isep.webapp.terasync.model.query.update.ValueByField;
import isep.webapp.terasync.repository.FaqRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.transaction.annotation.Transactional;

@Service
public class FaqService extends EntityService<Faq, FaqRepository> {
    @Autowired
    public FaqService(FaqRepository entityRepository) {
        super(entityRepository);
    }

    @Transactional
    @Override
    public Integer deleteEntityById(Integer id) {
        return entityRepository.deleteEntityById(id);
    }
}

File: front/src/app/service/faq.service.ts
import {Injectable} from '@angular/core';
import {HttpClient} from '@angular/common/http';
import {EntityService} from './entity.service';
import {Faq} from "../model/faq";

@Injectable({
  providedIn: 'root'
})
export class FaqService extends EntityService<Faq> {

  constructor(http: HttpClient) {
    super(http, "faq");
  }

}
